src/main/java/vazkii/pillar/StructureLoader.java

```diff
@@ -126,5 +127,41 @@ public static String getStructureNBTLocation(String jsonFileName) {
 	public static String jsonifySchema(StructureSchema schema) {
 		return gson.toJson(schema);
 	}
-	
+
+	public static StructureSchema getDefaultSchema() {
```

Comment:
The generatorType being blank is intentional, so you fill in your own. The trailing , I should fix though.